posydon/visualization/plot_pop.py

```diff
@@ -251,4 +258,137 @@ def plot_hist_properties(x, df_intrinsic=None, df_observable=None,
     if path:
         plt.savefig(path)
     if show:
-        plt.show()
\ No newline at end of file
+        plt.show()
+        
+        
+def plot_popsyn_over_grid_slice(pop, grid_type, met_Zsun, slices=None, channel=None, 
+                                plot_dir='./', prop=None, prop_range=None, 
+                                log_prop=False, alpha=0.3, s=5., 
+                                show_fig=True, save_fig=True, close_fig=True,
+                                verbose=False):
+
```

Comment:
Please add doc-strings to all new defined functions, which explain the function, the variables, and the return. Having a tutorial here is nice, but in a year the tutorial will be forgotten, while the code remains.

Reply:
The entire plotting file methods needs docstrings. I think we can postpone this to a later stage as all these methods are evolving quickly and might now be the final product.